components/Table/Search.tsx

```diff
@@ -38,7 +38,7 @@ const StyledSearchInput = styled(SearchInput)`
 	height: 100%;
 	text-indent: 16px;
 	border-radius: 8px;
-	padding: 10px 10px;
+	padding: 10px 15px;
```

Comment:
The space between the magnifier icon and placeholder `Search...` is a little bit narrow. So I add more space.

**Before**
![image](https://user-images.githubusercontent.com/4819006/164997673-5cded55a-9e88-4d43-9ce0-17e7d1686e3b.png)
**After**
![image](https://user-images.githubusercontent.com/4819006/164997671-918a728e-17d7-4778-850d-f8e76eed3ff6.png)